types/has/package.json

```diff
@@ -0,0 +1,7 @@
+{
+    "private": true,
+    "types": "index",
+    "typesVersions": {
+        ">=3.1.0-0": { "*": ["ts3.1/*"] }
```

Comment:
The only difference is `any` vs `unknown` in the mapped type, right? Then I don't think you need a separate 3.1 version since 3.0 supports `unknown` too.

Reply:
Right, I hadn’t noticed that the minimum supported **TypeScript** version was bumped to **3.0**.